plover/dictionary/test_default_dict.py

```diff
@@ -4,7 +4,7 @@
 import json
 import unittest
 
-DICT_PATH='plover/assets/dict.json'
+DICT_PATH='plover/assets/dict_unsorted.json'
```

Comment:
True. I just figured it was worth a quick fix so at least the tests don't error.